docs/ops.html

```diff
@@ -1129,6 +1128,26 @@ <h4><a id="remote_jmx" href="#remote_jmx">Security Considerations for Remote Mon
         <td>kafka.server:type=BrokerTopicMetrics,name=ReassignmentBytesInPerSec</td>
         <td></td>
       </tr>
+      <tr>
+        <td>Size of a partition on disk (in bytes)</td>
+        <td>kafka.log:type=Log,name=Size,topic=([-.\w]+),partition=([0-9]+)</td>
+        <td>The size of a partition on disk, measured in bytes.</td>
+      </tr>
+      <tr>
+        <td>Number of log segments in a partition</td>
+        <td>kafka.log:type=Log,name=NumLogSegments,topic=([-.\w]+),partition=([0-9]+)</td>
+        <td>The number of log segments in a partition.</td>
```

Comment:
I wasn't sure what information to add for this column of "normal value". 
* For date-based retention topics, it should roughly be sizeOfTopic/segmentSize
* But for compacted topics, that isn't the case
* If this is too high, it will mean more file handles.
* If it is too high, it might inefficient (is this true?)

Anyway, I wasn't sure what to put, so I left it simple.

Reply:
I think this is fine :)